ac/ac-chat/src/Dashboard.jsx

```diff
@@ -2,26 +2,32 @@
 
 import React from 'react';
 import WordCloud from 'react-d3-cloud';
+import { type LogDBT } from 'frog-utils';
 
 const fontSizeMapper = word => Math.max(10, Math.min(word.value * 10, 250));
 
-export default ({ logs }: any) => {
-  const data = {};
-  logs.forEach(log => {
-    log.chat.split(' ').forEach(word => {
-      data[word] = (data[word] || 0) + 1;
-    });
-  });
+const Viewer = ({ data }: Object) => (
+  <div>
+    <p>{data.length} data</p>
+    <WordCloud
+      data={[...Object.keys(data).map(w => ({ text: w, value: data[w] }))]}
+      fontSizeMapper={fontSizeMapper}
```

Comment:
Could the font size mapper be smarter? 
Right now we reach the maximum size when one word is repeated 25 times. That will happen often with large classrooms. The size of one word repeated 25 times should decrease if there are other word repeated 100 times.